app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find_by(id: params[:id])
+    if @quote.nil?
+      render :error
```

Comment:
to clarify @Murph33 I was suggesting using `find!` and letting rails render the 404 automatically